snapshots/smoking/2024-05-30/cigarette_sales.zip.dvc

```diff
@@ -0,0 +1,28 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: International Smoking Statistics
+    date_published: "2016-12-19"
+    title_snapshot: International Smoking Statistics - Cigarette Sales
```

Comment:
The field `title_snapshot` is only necessary if there are multiple snapshots for the same data product.
For example, [the Global Carbon Budget](https://github.com/owid/etl/tree/master/snapshots/gcp/2023-12-12) has multiple files, so the title of all snapshot files is `Global Carbon Budget`, but each file has a `title_snapshot`, e.g. `Global Carbon Budget - Fossil CO2 emissions`.
In this case, as far as I can tell, there is only one (and I suppose we are not expecting to create more). So `title_snapshot` is not required. Alternatively, you could create a file for each country, but that would be too many files, which is annoying, so a zip file is a much better choice (as you did).